packages/p2p/src/components/app-content.jsx

```diff
@@ -37,6 +39,20 @@ const AppContent = () => {
         return <AdvertiserPage />;
     }
 
+    if (general_store.props.setP2POrderProps) {
```

Comment:
```suggestion
    if (typeof general_store.props.setP2POrderProps === 'function') {
```
Or typeof `action`. To be very sure that u can actually invoke it